### PR TITLE
[LanguageSelector] - add component

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.atlassian.net/browse/SOV-788

### CODE REVIEW

**Comment by Rickk137** [comment]
@pietro-maximoff Is it possible to close dropdown once language is selected?

File: apps/frontend/src/app/2_molecules/LanguageSelector/LanguageSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      setCurrentLang(language);
+    },
+    [],
+  );
+
+  return (
+    <Dropdown
+      text={currentLang.toUpperCase()}
+      className={classNames(className)}
+      {...applyDataAttr(dataLayoutId)}
```
Comment (Rickk137) [inline apps/frontend/src/app/2_molecules/LanguageSelector/LanguageSelector.tsx]: size={DropdownSize.small}

**Review by Rickk137 — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/2_molecules/LanguageSelector/LanguageSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React, { FC, useCallback, useState } from 'react';
+
+import classNames from 'classnames';
+import i18next from 'i18next';
+import { reactLocalStorage } from 'reactjs-localstorage';
+
+import { Menu, MenuItem } from '@sovryn/ui/src/2_molecules';
+import { Dropdown, DropdownSize } from '@sovryn/ui/src/2_molecules/Dropdown';
+import { applyDataAttr } from '@sovryn/ui/src/utils';
```
Comment (soulBit) [inline apps/frontend/src/app/2_molecules/LanguageSelector/LanguageSelector.tsx]:
```suggestion
import {
  Menu,
  MenuItem,
  Dropdown,
  DropdownSize,
  applyDataAttr,
} from '@sovryn/ui';
```

File: yarn.lock [hunk 3]
```diff
@@ -4676,6 +4676,11 @@
     "@types/scheduler" "*"
     csstype "^3.0.2"
 
+"@types/reactjs-localstorage@^1.0.0":
```
Comment (soulBit) [inline yarn.lock]:
```suggestion
"@types/reactjs-localstorage@1.0.0":
```
Run `yarn` again and lockfile should be updated with this new dependency version

**Review by soulBit — CHANGES_REQUESTED** [review]

**Review by soulBit — APPROVED** [review]

**Review by Rickk137 — APPROVED** [review]

**Pull request merged**